benchmark_runner/main/main.py

```diff
@@ -8,7 +8,7 @@
 from benchmark_runner.main.environment_variables import environment_variables
 from benchmark_runner.common.clouds.Azure.azure_operations import AzureOperations
 from benchmark_runner.common.clouds.IBM.ibm_operations import IBMOperations
-
+from benchmark_runner.clusterbuster.clusterbuster_workloads import ClusterBusterWorkloads
```

Comment:
Group this line with the other workloads (after line 7).

Reply:
Fixed